Retesteth/eth/eth.sh

```diff
@@ -0,0 +1,2 @@
+echo "Starting $1 $2"
+eth --test --db-path $1 --ipcpath $2 --log-verbosity 4
```

Comment:
Just writing up a changelog for a new release (will make a draft first), is this for internal usage or is it meant to be documented?

What is this ``eth`` tool generally doing?